src/Trace/Exporter/ZipkinExporter.php

```diff
@@ -63,19 +54,15 @@ class ZipkinExporter implements ExporterInterface
      * Create a new ZipkinExporter
      *
      * @param string $name The name of this application
-     * @param string $host The hostname of the Zipkin server
-     * @param int $port The port of the Zipkin server
-     * @param string $endpoint (optional) The path for the span reporting
-     *        endpoint. **Defaults to** `/api/v2/spans`
-     * @param array $server (optoinal) The server array to search for the
+     * @param string $endpointUrl (optional) The url for the span reporting
+     *        endpoint. **Defaults to** `http://localhost:9411/api/v2/spans`
+     * @param array $server (optional) The server array to search for the
      *        SERVER_PORT. **Defaults to** $_SERVER
      */
-    public function __construct($name, $host, $port, $endpoint = '/api/v2/spans', $server = null)
+    public function __construct($name, $endpointUrl = null, array $server = null)
```

Comment:
Just note that this is a breaking change if anyone is using this client.

Reply:
Please do a new release if possible.